.github/workflows/CI.yml

```
@@ -0,0 +1,50 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags: '*'
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '0.7'
+          - '1.0'
+          - '1.1'
+          - '1.2'
+          - '1.3'
+          - '1.4'
```

```suggestion
          - '1.6'
          - '1.7'
          - '1.8'
          - '1.9'
```

---

Alternatively just test on the first and last supported version to save time
```suggestion
          - '1.6'
          - '1'
```